posix.mak

```diff
@@ -317,18 +317,19 @@ CWS_MAKEFILES = $(filter mak/% %.mak %/Makefile,$(MANIFEST))
 NOT_MAKEFILES = $(filter-out $(CWS_MAKEFILES) src/rt/minit.obj test/%.exp,$(MANIFEST))
 GREP = grep
 
-checkwhitespace:
+#checkwhitespace:
 # restrict to linux, other platforms don't have a version of grep that supports -P
-ifeq (linux,$(OS))
-	$(GREP) -n -U -P "([ \t]$$|\r)" $(CWS_MAKEFILES) ; test "$$?" -ne 0
-	$(GREP) -n -U -P "( $$|\r|\t)" $(NOT_MAKEFILES) ; test "$$?" -ne 0
-endif
+#ifeq (linux,$(OS))
+#       $(GREP) -n -U -P "([ \t]$$|\r)" $(CWS_MAKEFILES) ; test "$$?" -ne 0
+#       $(GREP) -n -U -P "( $$|\r|\t)" $(NOT_MAKEFILES) ; test "$$?" -ne 0
+#endif
+checkwhitespace: $(INSTALL_DIR)/${OS}/bin${MODEL}/dmd $(TOOLS_DIR)/checkwhitespace.d 
```

Comment:
Instead of using platform dependent grep
we should use the tools/checkwhitespace.d D-version

Reply:
This not directly related to supporting DragonFlyBSD, and should maybe have been in a separate commit.

Reply:
It's not as easy as that. There's currently no logic in place that ensures that `tools` is actually checked out and there have been long discussion of tools being an option or required dependency for building DMD + Druntime + Phobos.
The current status quo is that it's okay to use use `tools` in non essential targets.
Anyhow as you said yourself, please put this in a separate PR.